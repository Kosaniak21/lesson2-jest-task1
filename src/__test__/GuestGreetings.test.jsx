import React from "react";
import { shallow } from "enzyme";
import GuestGreeting from "./../GuestGreeting.jsx";
describe("GuestGreeting", () => {
  it(`should display guest greeting`, () => {
    const component = shallow(<GuestGreeting />);
    expect(component).toMatchSnapshot();
  });
});
