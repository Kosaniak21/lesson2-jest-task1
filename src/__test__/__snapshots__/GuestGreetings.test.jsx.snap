// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GuestGreeting should display guest greeting 1`] = `ShallowWrapper {}`;
